### ISSUE TITLE
Accessibility Feedback for Staging Review - Find a VA Form Post-Launch MVP

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18387]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

VSA: Josh
VSP: Trevor

## Details

This epic contains the accessibility feedback for Staging Review - Find a VA Form Post-Launch MVP.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Access Information

Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_

* Staging url: https://staging.va.gov/find-forms/

* Test cases (itemized steps to test): [VSA TestRail test cases](https://dsvavsp.testrail.io/index.php?/suites/view/136&group_by=cases:section_id&group_id=824&group_order=asc)

**Comment by joshkimux** [comment]
I don't think this is a part of the scope of this actual work, but I think it's worth pointing out. Currently, lists are not being announced as lists through VoiceOver which is probably due to the styling being removed e.g. `list-style:none` . I tested around, but adding `role="list"` and `role="listitem"` didn't seem to work. @1Copenut , do you have any history on this?

<img width="1340" alt="Screen Shot 2021-01-12 at 11 19 22 AM" src="https://user-images.githubusercontent.com/14154792/104343220-ac408000-54c9-11eb-909d-2e5e43a715e5.png">

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @joshkimux!  Good find.

**Comment by joshkimux** [comment]
@MarciMcGuireGCIO I'll be moving that one into a ticket! Looks like it was a part of the test cases! 🙇

**Comment by 1Copenut** [comment]
> I tested around, but adding `role="list"` and `role="listitem"` didn't seem to work. @1Copenut , do you have any history on this?

@joshkimux Adding a `role="list"` should restore the list items being announced as such. VO sometimes doesn't honor changes in the developer tools. Go ahead and log this as an issue, because it's worth improving.

**Comment by joshkimux** [comment]
@1Copenut I've completed this staging review using Firefox/NVDA, JAWs/Chrome, and Safari/VoiceOver! @MarciMcGuireGCIO and @zacharymorel have also conducted basic accessibility tests as well. This should be ready for VSP review.

**Comment by 1Copenut** [comment]
Thank you @joshkimux! We'll give it a look and follow up with you if any questions come up before the Thursday staging review.

**Comment by MarciMcGuireGCIO** [comment]
@joshkimux and @1Copenut, thank you so much for your reviews!

**Comment by MarciMcGuireGCIO** [comment]
@joshkimux , @1Copenut -- Reading through the accessibility issues, I think that most of them were recorded as part of the MVP. If I determine they are duplicates, can I close? Thx!

**Comment by joshkimux** [comment]
@MarciMcGuireGCIO That sounds fair to me! @1Copenut what are your thoughts?

**Comment by 1Copenut** [comment]
Agreed, if there are duplicates, please feel free to close!

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.
